src/packs/journals/Combat_wnqp2PFeCN9C0VOw/journal_Conditions_hDhdILCi65wpBgPZ.json

```diff
@@ -257,12 +257,12 @@
       "_stats": {
         "compendiumSource": null,
         "duplicateSource": null,
-        "coreVersion": "13.341",
+        "coreVersion": "13.346",
         "systemId": "draw-steel",
         "systemVersion": "0.1.0",
         "createdTime": null,
-        "modifiedTime": 1735075928328,
-        "lastModifiedBy": "cpRCYfteS3IEfu8O",
+        "modifiedTime": 1754089640327,
+        "lastModifiedBy": "TJSowGV8FgjirefO",
```

Comment:
We should reduce the diff changing the unpacking script to null'ing `modifiedTime` and inserting a placeholder id for `lastModifiedBy` - something like `drawsteelbuilder` (exactly 16 characters as it happens)